concent_api/core/tests/test_integration_accept_or_reject_results.py

```diff
@@ -1794,43 +1808,27 @@ def test_requestor_send_again_subtask_results_accepted_or_rejected_when_message_
             )
         )
 
-        self._store_golem_messages_in_database(
-            message_type            = deserialized_force_subtask_results.TYPE,
-            timestamp               = "2018-02-05 10:00:30",
-            data                    = deserialized_force_subtask_results,
-            task_id                 = deserialized_force_subtask_results.ack_report_computed_task.report_computed_task.task_to_compute.compute_task_def['task_id'],  # pylint: disable=no-member
-        )
-        self._assert_stored_message_counter_increased()
-
-        deserialized_subtask_results_response = self._get_deserialized_force_subtask_results_response(
-            timestamp = "2018-02-05 11:00:00",
-            subtask_results_rejected = self._get_deserialized_subtask_results_rejected(
-                timestamp   = "2018-02-05 11:00:00",
-                reason      = message.tasks.SubtaskResultsRejected.REASON.VerificationNegative,
-                report_computed_task = self._get_deserialized_report_computed_task(
-                    subtask_id = '2',
-                    task_to_compute = self._get_deserialized_task_to_compute(
-                        timestamp   = "2018-02-05 11:00:00",
-                        deadline    = "2018-02-05 11:00:05",
-                        task_id     = '2',
-                    )
-                )
-            )
-        )
-
-        self._store_golem_messages_in_database(
-            message_type            = deserialized_subtask_results_response.TYPE,
-            timestamp               = "2018-02-05 11:00:01",
-            data                    = deserialized_subtask_results_response,
-            task_id                 = deserialized_subtask_results_response.subtask_results_rejected.report_computed_task.task_to_compute.compute_task_def['task_id'],  # pylint: disable=no-member
-        )
-        self._assert_stored_message_counter_increased()
-
         compute_task_def = self._get_deserialized_compute_task_def(
             task_id     = '2',
             deadline    = "2018-02-05 11:00:00",
         )
 
+        subtask = store_or_update_subtask(
+            task_id=deserialized_force_subtask_results.ack_report_computed_task.report_computed_task.task_to_compute.compute_task_def['task_id'],  # pylint: disable=no-member,
+            subtask_id=deserialized_force_subtask_results.ack_report_computed_task.report_computed_task.task_to_compute.compute_task_def['subtask_id'],  # pylint: disable=no-member,
+            provider_public_key=self.PROVIDER_PUBLIC_KEY,
+            requestor_public_key=self.REQUESTOR_PUBLIC_KEY,
+            state=Subtask.SubtaskState.ACCEPTED,
+            report_computed_task=deserialized_force_subtask_results.ack_report_computed_task.report_computed_task,  # pylint: disable=no-member,
+            task_to_compute=deserialized_force_subtask_results.ack_report_computed_task.report_computed_task.task_to_compute,  # pylint: disable=no-member,
```

Comment:
please extract `task_to_compute` and use it also above to obtain `task_id` and `subtask_id`

Reply:
:+1: done